### PR TITLE
Update Linux app details to remove Google mention

### PR DESCRIPTION
**Pull request by bsclifton**
(NOTE: I haven't built/tested on Linux yet; will grab binaries from PR builder)

Please take a look at the files edited. Do these edits look correct? I want to make sure we update our Brand details (and also have relevant links for product issues / support), but I don't want to break any licensing terms

Fixes https://github.com/brave/brave-browser/issues/392

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Test Plan:
See https://github.com/brave/brave-browser/issues/392

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

**Comment by bsclifton** [comment]
There's a 3rd file at `src/chrome/installer/linux/common/google-chrome/google-chrome.info` which has more details, but that looks to be Chrome specific (ex: not Chromium)

File: patches/chrome-installer-linux-common-google-chrome-google-chrome.appdata.xml.template.patch [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+index 828dc1f0b625f411538751a8f8a12088aed7d729..b11609c809cc7931c512e3c30be911d82db66fa7 100644
+--- a/chrome/installer/linux/common/google-chrome/google-chrome.appdata.xml.template
++++ b/chrome/installer/linux/common/google-chrome/google-chrome.appdata.xml.template
+@@ -2,11 +2,11 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+   <id>@@PACKAGE@@.desktop</id>
+-  <update_contact>chromium-dev@chromium.org</update_contact>
++  <update_contact>support@brave.com</update_contact>
+   <metadata_license>CC0-1.0</metadata_license>
```
Comment (bsclifton) [inline patches/chrome-installer-linux-common-google-chrome-google-chrome.appdata.xml.template.patch]: NOTE: this `metadata_license` is for the actual XML file itself (not the project). More info available at https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by simonhong** [comment]
Tested on my local machine and manpage patch works as expected.
Regarding to `appdata`, we don't use `chrome.appdata.xml.template`. 
Instead, we override `chromium-browser.appdata.xml` in `chrome/installer/linux/BUILD.gn`.
So, how about overriding `manpage.1.in` instead of patching like `appdata.xml` did?

**Comment by bsclifton** [comment]
@simonhong good call- will update to do that 😄

**Comment by bsclifton** [comment]
@simonhong @iefremov updated 👍

**Comment by bsclifton** [comment]
Verified this looks great by downloading the binaries provided by PR builder (https://s3-us-west-2.amazonaws.com/brave-jenkins-build-artifacts/brave-browser-build-pr/update-linux-app-details/3/brave-browser-dev_0.64.4_amd64.deb)

## Before
Screen Shot 2019-03-18 at 1 56 16 PM
Screen Shot 2019-03-18 at 1 56 35 PM


## After
Screen Shot 2019-03-18 at 1 56 48 PM
Screen Shot 2019-03-18 at 1 57 05 PM

**Pull request merged**